core/src/apps/management/apply_settings.py

```diff
@@ -69,16 +71,16 @@
     await require_confirm(ctx, text, ButtonRequestType.ProtectCall)
 
 
-async def require_confirm_change_passphrase_source(ctx, source):
-    if source == PassphraseSourceType.DEVICE:
-        desc = "ON DEVICE"
-    elif source == PassphraseSourceType.HOST:
-        desc = "ON HOST"
-    else:
-        desc = "ASK"
+async def require_confirm_change_passphrase_source(
+    ctx, passphrase_always_on_device: bool
+):
     text = Text("Passphrase source", ui.ICON_CONFIG)
-    text.normal("Do you really want to", "change the passphrase", "source to")
-    text.bold("ALWAYS %s?" % desc)
+    if passphrase_always_on_device:
+        text.normal(
+            "Do you really want to", "entry passphrase always", "on the device?"
```

Comment:
typo: "enter"

but the wording is not great either. let's have someone from a more wording-y department look this over?

Reply:
Yes. I've mentioned this to @matejzak, he made a note that he will address it when they do copy in Suite.